admin/views/dashboard-widget.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * @package WPSEO\Admin
+ *
+ * @var array $statistics {
+ *      An array of statistics to display
+ *
+ *      @type string $seo_rank The SEO rank that this item represents
+ *      @type string $title The title for this statistic
+ *      @type string $class The class for the link
+ *      @type int $count The amount of posts that meets the statistic requirements
+ * }
+ */
+
+?>
+<p><?php _e( 'Below are your published posts&#8217; SEO scores. Now is as good a time as any to start improving some of your posts!', 'wordpress-seo' ); ?></p>
```

Comment:
> this is the correct way to add quotes in translations

Documented source for that statement? I'd rather see the translators lives made a little easier by having more readable English strings (as it may be their second or third language anyway), with standard apostrophe, and `wptexturise()` applied where needed.